Funcion agregar 5 de oros y arraylists de palos

Sakerick opened the pull request:
Se agrega la función para que el jugador que tenga el 5 de oros lo coloque en su respectivo arraylist y sea el que comience el juego. Tambien se crearon los arraylists para cada palo.

Sakerick committed:
Funcion agregar 5 de oros

Se agrega la función para que el jugador que tenga el 5 de oros lo coloque en su respectivo arraylist y sea el que comience el juego. Tambien se crearon los arraylists para cada palo

Mulansito-l commented on app/src/main/java/elCinquillo.java:
```diff
@@ -6,12 +6,20 @@ public class elCinquillo {
         barajaParaJugar = new Baraja();
+        ArrayDeOros=new ArrayList<Carta>();
+        ArrayDeBastos=new ArrayList<Carta>();
+        ArrayDeOros=new ArrayList<Carta>();
```
Creaste 2 arraylist de oros.

Mulansito-l commented on app/src/main/java/elCinquillo.java:
```diff
@@ -57,4 +67,19 @@ public void mostrarManosDeJugador() {
             i++;
         }
     }
+    public void agregar5AlCentro() {
```
No se agrega el 5 chabalin, imprimí el array de oros y no muestra que tenga algo dentro :c.

Mulansito-l commented on app/src/main/java/elCinquillo.java:
```diff
@@ -23,6 +31,8 @@ public void jugarElCinquillo() {
         repartirCartas();
         mostrarManosDeJugador();
+        agregar5AlCentro();
+        System.out.println(ArrayDeOros);
```
El array está vacio.

Mulansito-l reviewed (changes requested):
Checa lo que puse en los comments chabal.

Sakerick committed:
Correccion de la funcion para agregar el 5

Hacia falta que la variable j se volviera a inicializar en 0 despues de recorrer las cartas de la mano del primer jugador. Tambien se eliminó el array de oros repetido

Mulansito-l reviewed (approved):
Ya funciona correctamente, muy bien chabalin 👍 .
